projects/Panoptic-DeepLab/README.md

```diff
@@ -11,8 +11,17 @@ Bowen Cheng, Maxwell D. Collins, Yukun Zhu, Ting Liu, Thomas S. Huang, Hartwig A
 ## Installation
 Install Detectron2 following [the instructions](https://detectron2.readthedocs.io/tutorials/install.html).
 
-## Training
+If using the cityscape, follow [these instructions](https://detectron2.readthedocs.io/tutorials/builtin_datasets.html#expected-dataset-structure-for-cityscapes) to download the data set and convert it to panoptic.
+
+If you are running on a CPU, update the device in the models [config yaml](github.com/facebookresearch/detectron2/blob/master/projects/Panoptic-DeepLab/configs/Cityscapes-PanopticSegmentation/Base-PanopticDeepLab-OS16.yaml) with the following:
+```yaml
+# /projects/Panoptic-DeepLab/configs/Cityscapes-PanopticSegmentation/Base-PanopticDeepLab-OS16.yaml
+_BASE_: Base-PanopticDeepLab-OS16.yaml
+MODEL:
+  DEVICE: 'cpu'
+```
```

Comment:
Can you remove this part? There are many useful configs that are universally applicable to every model. We are not documenting every one of them for every model.